components/kyma-environment-broker/internal/migrations/instancedetails_test.go

```diff
@@ -44,35 +45,10 @@ func TestInstanceDetailsMigration_Migrate(t *testing.T) {
 }
 
 func fixProvisioningOperation() internal.ProvisioningOperation {
-	return internal.ProvisioningOperation{
-		Operation: internal.Operation{
-			InstanceDetails: internal.InstanceDetails{
-				Lms: internal.LMS{
-					TenantID: "lms-tenant",
-				},
-				SubAccountID: "subacc-id",
-				RuntimeID:    "runtime-id",
-				ShootName:    "shoot-name",
-				ShootDomain:  "shoot-domain",
-				XSUAA: internal.XSUAAData{
-					Instance: internal.ServiceManagerInstanceInfo{
-						BrokerID:              "broker-id",
-						ServiceID:             "service-id",
-						PlanID:                "plan-id",
-						InstanceID:            "instance-id",
-						Provisioned:           true,
-						ProvisioningTriggered: true,
-					},
-					XSAppname: "xsapp-name",
-					BindingID: "binding-id",
-				},
-			},
-			ID:         "prov-op-id",
-			Version:    0,
-			CreatedAt:  time.Now(),
-			UpdatedAt:  time.Now().Add(1 * time.Minute),
-			InstanceID: "instance-id",
-			State:      domain.Succeeded,
-		},
-	}
+	provisioningOperation := fixture.FixProvisioningOperation("prov-op-id", "instance-id")
+	provisioningOperation.Operation.InstanceDetails = fixture.FixInstanceDetails("id")
```

Comment:
Is there any reason why we do not set `InstanceDetails` when creating fixed operation in `FixOperation` function? In real flow, the `InstanceDetails` field is always somehow populated into the struct, because we unmarshall `Data` field from the DB (see `postsql/operation.go`). I've checked and it probably breaks some assertions connected with AVS evaluation like:
```golang
assert.Contains(t, mockAvsSvc.evals, inDB.Avs.AVSEvaluationExternalId)
```

this is why the fixture `InstanceDetails` probably should have `Avs` struct empty only in special cases like testing steps which touch AVS evaluation, but globally whole `InstanceDetails` struct should be filled in `Operation` fixture.

Reply:
InstanceDetails will be populated in Instance & Operation fixtures.